stable/xray/README.md

```diff
@@ -357,6 +357,7 @@ The following table lists the configurable parameters of the xray chart and thei
 | `indexer.name`                                 | Xray Indexer name                            | `xray-indexer`       |
 | `indexer.image`                                | Xray Indexer container image                 | `docker.bintray.io/jfrog/xray-indexer`  |
 | `indexer.replicaCount`                         | Xray Indexer replica count                   | `1`                  |
+| `indexer.annotations`                          | Xray Indexer annotations                     | `{}`                               |
```

Comment:
What about analysis.annotations ?
I see you added that to the statefulset but it’s missing from the readme

Reply:
Resolved!